smt/pkg/smt/smt_batch.go

```diff
@@ -20,11 +20,7 @@ func (s *SMT) InsertBatch(ctx context.Context, logPrefix string, nodeKeys []*uti
 	var smtBatchNodeRoot *smtBatchNode
 	nodeHashesForDelete := make(map[uint64]map[uint64]map[uint64]map[uint64]*utils.NodeKey)
 
-	//start a progress checker
-	preprocessStage := uint64(size) / 10
```

Comment:
I think we lost the printing for preprocess stage. I suppose it was intentional.